src/13_file_io.py

```diff
@@ -9,10 +9,16 @@
 # Print all the contents of the file, then close the file
 
 # YOUR CODE HERE
+with open('foo.txt') as f:
+    read_data = f.read()
+    print(read_data)
 
```

Comment:
Great use of `with open()`. I pretty mush use this exclusively for file IO as the file doesn't need to be explicitly closed.